README.md

```diff
@@ -34,112 +42,64 @@ window.__env.userLevels.regular = /*Integer containing the required user level f
-- `docker-compose-host-nginx.yml` is a variant of the above but the nginx must run on the host and is not run in a container.
-- `docker-compose-local.yml` starts all microservices as containers that were build locally. Only useful for testing container-packaging of apps. The microservice image names are simply the name without leading `o2r-`, so `muncher`, `bouncer`, etc. The client is mounted from the host, see above.
-- `docker-compose-local-platformcontainer.yml` the same as the previous configuration, but the client is also started in a container based on the local image named `platform`.
+## Development environment with Docker Compose
 
-The configurations all use a common volume `o2r_test_storage` (with the global name `test_o2r_test_storage` because the name of the directory of this file is preprended by Docker), and a common network `o2rnet` (with the global name `test_o2rnet`).
+You can start all required o2r microservices (using latest images from [Docker Hub](https://hub.docker.com/r/o2rproject)) with just two commands using `docker-compose` (version `1.9.0+`) and Docker (version `1.13.0+`).
 
-The volume and network can be inspected for development purposes:
+First, **read the instructions on preparing your host machine in the [`reference-implementation`](https://github.com/o2r-project/reference-implementation) project**.
 
-```bash
-docker volume ls
-docker volume inspect test_o2r_test_storage
-docker network ls
-docker network inspect test_o2rnet
-```
-
-You can remove the storage volumes by running `docker-compose down -v`.
-
-#### Host preparation
-
-Elasticsearch requires the ability to create many memory-mapped areas ([mmaps](https://en.wikipedia.org/wiki/Mmap)s) for fast access. The usual max map count check setting is [configured to low on many computers](https://www.elastic.co/guide/en/elasticsearch/reference/5.0/_maximum_map_count_check.html). You must configure `vm.max_map_count` on the host to be at least `262144`, e.g. on Linux via `sysctl`. You can find instructions for all hosts (including Docker Toolbox) in the [Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/5.0/docker.html#docker-cli-run-prod-mode).
+This project contains one `docker-compose` configuration (file `docker-compose.yml`) to run all microservices & databases, and mount the client application directly from the source directory `client`.
 
-#### Required settings
+_The client must be build on the host!_
 
-Some of the settings to run the platform cannot be published. These must be provided at runtime using envionment variables as is described in the OS-specific instructions below. Not providing one of these paramters results in untested behaviour.
+**Required settings**
 
+Some of the settings to run the platform cannot be published.
```

Comment:
Refer to git ignore?

Reply:
I do not understand. Do you mean to add these settings to a file which is already included in the gitignore?

Reply:
I think there is a misunderstanding. The text does not refer to settings stored in a file but as environment variables. Therefore gitignore does not need to be changed.